peer_mentoring/account_management/views.py

```diff
@@ -20,11 +20,9 @@ def profile_home(request):
     form = CustomUserChangeForm()
     photo_upload = AddPhotoForm()
     # user_photos = Photo.objects.filter(user=user)
-    recent_posts = (
-        Post.objects.filter(author=request.user).order_by("added").reverse()[0:9]
-    )
+    recent_posts = Post.objects.filter(author=user).order_by("added").reverse()[0:9]
```

Comment:
[:10] rather? You can also drop reverse(), just use `-added`

Reply:
you can also made the ordering in descended order the default by setting this on your model:

```
    class Meta:
        ordering = ['-added']
```